### PR TITLE
Updated spatial_transformer.py to include `tf.meshgrid`

### PR DESCRIPTION
**Pull request by parambharat**
Changed the `_meshgrid` function to use the new `tf.meshgrid` and `tf.linspace` available from tensorflow rc.0.10 +

### CODE REVIEW

**Comment by nealwu** [comment]
@daviddao could you take a look at this?

File: transformer/spatial_transformer.py [hunk 1]
```diff
@@ -128,13 +128,8 @@ def _meshgrid(height, width):
...
             #  grid = np.vstack([x_t.flatten(), y_t.flatten(), ones])
-            x_t = tf.matmul(tf.ones(shape=tf.pack([height, 1])),
-                            tf.transpose(tf.expand_dims(tf.linspace(-1.0, 1.0, width), 1), [1, 0]))
-            y_t = tf.matmul(tf.expand_dims(tf.linspace(-1.0, 1.0, height), 1),
-                            tf.ones(shape=tf.pack([1, width])))
-
-            x_t_flat = tf.reshape(x_t, (1, -1))
-            y_t_flat = tf.reshape(y_t, (1, -1))
+            
+            x_t, y_t = tf.meshgrid(tf.linspace(-1,1,width), tf.linspace(-1,1,height))
```
Comment (daviddao) [inline transformer/spatial_transformer.py]: Error: ```tf.linspace``` doesn't take int as start and end argument but floats

**Comment by daviddao** [comment]
@nealwu @parambharat unfortunately the PR breaks the examples provided. Would you mind fixing it?

**Comment by nealwu** [comment]
Closing old PRs.

**Pull request closed without merging**